.github/src/context.js

```diff
@@ -12,13 +12,30 @@ import { PER_PAGE_MAX } from "./github.js";
  * @returns {Promise<{owner: string, repo: string, head_sha: string, issue_number: number, run_id: number }>}
  */
 export async function extractInputs(github, context, core) {
-  core.info(`extractInputs(${context.eventName}, ${context.payload.action})`);
+  core.info("extractInputs()");
+  core.info(`  eventName: ${context.eventName}`);
+  core.info(`  payload.action: ${context.eventName}`);
+  core.info(
+    `  payload.workflow_run.event: ${context.payload.workflow_run?.event || "undefined"}`,
```

Comment:
Log `payload.workflow_run.event` by default, since it's very useful for debugging.